internal/wasm/jit/engine.go

```diff
@@ -809,10 +791,11 @@ func compileHostFunction(f *wasm.FunctionInstance) (*compiledFunction, error) {
 	}
 
 	return &compiledFunction{
-		source:             f,
-		codeSegment:        code,
-		codeInitialAddress: uintptr(unsafe.Pointer(&code[0])),
-		stackPointerCeil:   stackPointerCeil,
+		source:                f,
+		codeSegment:           code,
+		codeInitialAddress:    uintptr(unsafe.Pointer(&code[0])),
```

Comment:
curious.. I guess we don't need to do any `runtime.KeepAlive` stuff because compiledFunction holds a reference to we're getting the pointer to (and that means it won't move?)

Reply:
yes